### ISSUE TITLE
Disable notification `Makefile entry point not found`

### ISSUE THREAD

**Issue by alexreg** [microsoft/vscode-makefile-tools#394]
Is there any way to disable the notification `Makefile entry point not found`? I find myself encountering this quote often, since many projects only have makefiles in subdirectories, for specific tasks. Ideally the extension wouldn't activate at all in this case, but I'd settle for hiding this notification.

**Comment by bobbrow** [comment]
Thanks for reporting this.  I see this too.  We can make this better. I'll put it on the backlog for 0.7.

**Comment by alexreg** [comment]
@bobbrow Great, appreciate it.

**Comment by ushuz** [comment]
As a workaround, you can modify the extension code and disable the notification for now:

```js
// .vscode/extensions/ms-vscode.makefile-tools-0.6.0/out/src/configuration.js
...
        if (!util.checkFileExistsSync(makefileUsed)) {
            // vscode.window.showErrorMessage(localize(1, null, "Makefile"));
            logger.message("The makefile entry point was not found. " +
...
```

**Comment by andreeis** [comment]
@ushuz , @danielb2 , @alexreg , we removed the popup notifications regarding makefile and make missing, we left the logging about that unchanged and we are also showing now the "C/C++" tab UI on the left side even when before it was hidden. In there we add now some extra elements about status on make/makefile/build.log: path where we search for them, if not found, you can click the edit pen icon to open settings and change this state via settings like "makefile.makefilePath", "makefile.makePath", "makefile.buildLog" and their "makefile.configurations[]" counterparts.

This will be released with 0.7 next week but if you would like to see how this works earlier you can install one vsix from [this link](https://github.com/microsoft/vscode-makefile-tools/actions/runs/4451312188).

go to the Extensions panel on the left
click "..."
run "Install from vsix" from the context menu

Let us know if you encounter any other issues or if you have any other suggestions/recommendations in this area of functionality.

**Comment by alexreg** [comment]
@andreeis Okay, that sounds like a good solution. I'll have a look.
